k8s-ingress/ingress-controller.yaml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: l5d-system
```

Comment:
Sure I can move the namespace specification into the readme. Ingress controller examples I've seen either don't specify a namespace or straight up used `kube-system` as the namespace! That  seemed a little invasive to me.